SendWelcomeMessagesActivity/index.ts

```diff
@@ -175,20 +162,28 @@ const activityFunction: AzureFunction = async (
     return "FAILURE";
   }
 
+  if (!input.messageKind || !welcomeMessages[input.messageKind]) {
```

Comment:
this should be handled by a io-ts type that contains both messageKind and profile